src/types.js

```diff
@@ -458,6 +458,10 @@ export type UnreadState = {
   mentions: UnreadMentionsState,
 };
 
+/**
+ * Types the complete redux state. Each property corresponds to a top-level
```

Comment:
Should we even mention that it is 'typing' something? 
Also it looks like we should be able to say a bit more here...

Reply:
Yeah, I'm not thrilled about the `Typing` preamble either. I'd be glad about better ways to start these docstrings.

Reply:
What more would you say here? I avoided adding @prop for all the props in GlobalState, because they should be documented where the type is defined.